sound/soc/sof/intel/telemetry.h

```diff
@@ -0,0 +1,207 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ * telemetry data in debug windows
+ */
+
+#ifndef _SOF_INTEL_TELEMETRY_H
+#define _SOF_INTEL_TELEMETRY_H
+
+#define SOF_IPC4_FW_REPORTED_MAX_CORES_COUNT	5
+#define SOF_IPC4_FW_LP_TIMER_DELAY_CAUSE_COUNT	2
+#define SOF_IPC4_FW_MAX_CONDITION_DATA_COUNT	9
+#define SOF_IPC4_FW_MAX_NUMBER_MEM_BINS		1
+#define SOF_IPC4_FW_MAX_REPORTED_TASKS		40
+#define SOF_IPC4_FW_AREGS_COUNT			64
```

Comment:
all these values seem completely random to me.

Reply:
Refined but I don't know the idea for SOF_IPC4_FW_MAX_CONDITION_DATA_COUNT and SOF_IPC4_FW_MAX_REPORTED_TASKS